content/en/security/application_security/threats/add-user-info.md

```diff
@@ -720,12 +722,30 @@ In <code>extended</code> mode, the trace library tries to collect the user ID, a
 
 To configure automatic user event tracking mode, you can set the environment variable <code>DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING</code> to <code>safe</code> or <code>extended</code>. By default, the tracer library uses the <code>safe</code> mode.
 
+<div class="alert alert-info">For compatibility, see this <a href="/security/application_security/enabling/compatibility/">ASM capabilities table</a>.</div>
+
+Automatic user ID collection offers three modes: <code>identification</code>, <code>anonymization</code>, and <code>disabled</code>.
+
+To configure manually the automatic user ID collection mode, you can set the environment variable <code>DD_APPSEC_AUTO_USER_INSTRUMENTATION_MODE</code> to <code>ident</code>, <code>anon</code>, or <code>disabled</code>. By default, the tracer library uses the <code>ident</code> mode.
+
+You can configure the automatic user ID collection mode in your remote service configuration settings in the [Service Catalog][12]. Under the capabilities tab, you can choose the mode you want to apply.
+
+{{< img src="security/application_security/threats/service-capabilities-threat-management.png" alt="Access the Automated User ID Collection Mode Button from the Service Catalog by clicking on your service, then in the Capabilites tab" style="width:100%;" >}}
+
+The available modes are: <code>identification</code>, <code>anonymization</code>, <code>disabled</code> and <code>local</code>. Local mode removes all remote configuration and lets local configuration take over for the user ID collection mode.
+
 **Note**: There could be cases in which the trace library won't be able to extract any information from the user event. The event would be reported with empty metadata. In those cases, we recommend using the [SDK](#adding-business-logic-information-login-success-login-failure-any-business-logic-to-traces) to manually instrument the user events.
 
 ## Disabling automatic user activity event tracking
 
+<div class="alert alert-warning"><strong>This is deprecated</strong> and only applies to older agent versions before x</div>
+
 If you wish to disable the detection of these events, you should set the environment variable <code>DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING</code> to <code>disabled</code>. This should be set on the application hosting the Datadog Tracing Library, and not on the Datadog Agent.
 
+<div class="alert alert-info">For compatibility, see this <a href="/security/application_security/enabling/compatibility/">ASM capabilities table</a>.</div>
+
+If you wish to disable the automatic user ID collection, you should set the environment variable <code>DD_APPSEC_AUTO_USER_INSTRUMENTATION_MODE</code> to <code>disabled</code>. This should be set on the application hosting the Datadog Tracing Library, and not on the Datadog Agent. This can also be done by selecting <code>disabled</code> mode in remote service configuration settings.
```

Comment:
```suggestion
If you wish to disable the automatic user ID collection, you should set the environment variable <code>DD_APPSEC_AUTO_USER_INSTRUMENTATION_MODE</code> to <code>disabled</code>. This should be set on the application hosting the Datadog Tracing Library, and not on the Datadog Agent. This can also be done by selecting <code>disabled</code> mode in remote service configuration settings.

You should only turn this feature off, if you wish to prevent the collection of user IDs for privacy reasons. Disabling auto-instrumentation will prevent detection of some classes of attacks (such as account takeovers).
```